src/server/session.ts

```diff
@@ -294,71 +294,47 @@ namespace ts.server {
         return deduplicate(outputs, equateValues);
     }
 
-    type CombineOutputResult<T> = { project: Project; result: readonly T[]; }[];

```

Comment:
Most of these helpers were inlined.  The survivors were renamed and probably nested.